README.md

```diff
@@ -96,6 +96,12 @@ pip3 install poetry
 poetry install
 poetry shell
 alembic upgrade head
+
+cd openadapt/app/dashboard
+nvm install 21
+nvm use
+npm install
```

Comment:
Ideally all of this would be automated.

What do you think about something like this: https://stackoverflow.com/a/73649412

Reply:
```
./entrypoint.sh: line 3: nvm: command not found
./entrypoint.sh: line 4: nvm: command not found
```

```
# __init__.py
import os
import subprocess


def _run(bash_script):
    return subprocess.call(bash_script, shell=True)

def entrypoint():
    cwd = os.path.dirname(os.path.realpath(__file__))
    os.chdir(cwd)
    return _run(f"source ./entrypoint.sh")
```

```
# entrypoint.sh

#!/bin/bash
nvm install 21
nvm use
npm install
```

Reply:
Via ChatGPT:
```
#!/bin/bash
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion

nvm install 21
nvm use 21  # Specify the version to ensure it uses the right one
npm install
```

Reply:
Please also install npm and nvm in the installation scripts, and update the manual installation steps in the README to include this as well.

Reply:
Added on cfba05920595a3bfb92195088a9bf3e09f306f7b